test/utils/InstanceTest.sol

```diff
@@ -411,7 +427,7 @@ abstract contract InstanceTest is HyperdriveTest {
         uint256 aliceBalanceBefore = address(alice).balance;
 
         // Contribution in terms of base.
-        uint256 contribution = 5_000e18;
+        uint256 contribution = 1_000e18;
```

Comment:
why was this changed?

Reply:
This is too large of an amount of ETH pools, and it's just as good of a test, so I lowered it.